Libraries/Text/RCTShadowText.m

```diff
@@ -119,7 +119,7 @@ - (NSTextStorage *)buildTextStorageForWidth:(CGFloat)width
 
   NSTextContainer *textContainer = [NSTextContainer new];
   textContainer.lineFragmentPadding = 0.0;
-  textContainer.lineBreakMode = _numberOfLines > 0 ? NSLineBreakByTruncatingTail : NSLineBreakByClipping;
```

Comment:
@nicklockwood I just added those for you! :wink: I figured whoever maintains `Text` will understand it. Should I include them anyways?